src/applications/mhv-medical-records/actions/conditions.js

```diff
@@ -26,15 +39,28 @@ export const getConditionsList = (isCurrent = false) => async dispatch => {
 export const getConditionDetails = (
   conditionId,
   conditionList,
+  isAccelerating = false,
 ) => async dispatch => {
   try {
+    const getDetailsFunc = isAccelerating
+      ? async () => {
+          // Return a notfound response because the downstream API
+          // does not support fetching a single condition
+          return { data: { notFound: true } };
+        }
```

Comment:
How will this behave if you refresh the browser on the Details page? Will it fail due to notFound?

Reply:
Also I didn't see your [previous comment](https://github.com/department-of-veterans-affairs/vets-website/pull/38435#discussion_r2319351225), which stated this explicitly. 🙂